cmd/influx/main_test.go

```diff
@@ -236,3 +254,19 @@ func newNode(config *influxd.Config, path string) (*influxd.Node, error) {
 	}
 	return node, nil
 }
+
+func newPort() int {
+	l, e := net.Listen("tcp", ":0")
+	defer l.Close()
+	if e != nil {
+		panic(e)
+	}
+
+	parts := strings.Split(l.Addr().String(), ":")
+	p, err := strconv.Atoi(parts[len(parts)-1])
+	if err != nil {
+		panic(err)
+	}
+	return p
```

Comment:
When watching the ports log out it always went up continuously.  Could it create a race condition?  I'm not sure.  I suspect it has more to do with how the underlying platform implements it.  I will ask on Gopher slack though to see if anyone knows if there is some type of delay at the system level before it puts that port back on the market.